proms-resources/frontend/src/main/frontend/src/TOU.json

```
@@ -1,4 +1,80 @@
+  "text" : [
+    "These terms and conditions (the \"Agreement\") for a symptoms survey located at ePRO.uhn.ca and any associated website, portal, or similar service (collectively, the \"Application\"), constitute a legal agreement and is entered into by and between you (\"you\", \"your\") and the University Health Network (\"UHN\", \"we\", \"us\", \"our\").\\\n",
```

There should be no need for newline markers at the end of each array item, since we join by newline in the code. We can simply join by double newline instead.
```suggestion
    "These terms and conditions (the \"Agreement\") for a symptoms survey located at ePRO.uhn.ca and any associated website, portal, or similar service (collectively, the \"Application\"), constitute a legal agreement and is entered into by and between you (\"you\", \"your\") and the University Health Network (\"UHN\", \"we\", \"us\", \"our\").",
```

---

Done in 8c43d6e